From: "Barak A. Pearlmutter" <bap@debian.org>
Date: Sun, 1 Jan 2012 18:20:23 +0100
Subject: pointer type

quash gcc pointer type warning,
passing **dvd_reader_t to routine that expects *dvd_reader_t
---
 vobcopy.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vobcopy.c b/vobcopy.c
index d6e8ffc..95c3fea 100644
--- a/vobcopy.c
+++ b/vobcopy.c
@@ -773,7 +773,7 @@ and potentially fatal."  - Thanks Leigh!*/
 
   if( longest_title_flag ) /*no title specified (-n ) */
     {
-      titleid = get_longest_title( &dvd );
+      titleid = get_longest_title( dvd );
       fprintf( stderr, _("[Info] longest title %d.\n"), titleid );
     }
 
